### PR TITLE
[type:feature] Enhance metrics collection for Sentinel, Resilience4j, and Hystrix

### PR DESCRIPTION
**Pull request by GNK48-Ava**
Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by moremind** [comment]
pls resolve the conflict

**Comment by GNK48-Ava** [comment]
Conflict resolved. Pls review. Thank you~

**Comment by moremind** [comment]
looks good

**Comment by moremind** [comment]
pls polish the code

File: shenyu-plugin/shenyu-plugin-metrics/src/main/java/org/apache/shenyu/plugin/metrics/reporter/MetricsReporter.java [hunk 1]
```diff
@@ -68,80 +73,80 @@ public static void registerMetrics(final Collection<Metric> metrics) {
...
         }
     }
-    
+
     /**
      * Register counter.
      *
-     * @param name name
+     * @param name       name
      * @param labelNames label names
```
Comment (moremind) [inline shenyu-plugin/shenyu-plugin-metrics/src/main/java/org/apache/shenyu/plugin/metrics/reporter/MetricsReporter.java:81]: remove the blank

**Review by moremind — APPROVED** [review]

**Pull request merged**